server/src/appFactory.ts

```diff
@@ -16,14 +16,20 @@ const {
   SLACK_CHANNEL_LINK,
   SLACK_TARGET_CHANNEL,
   TRELLO_BOARD_LINK,
+  SERVER_URL
 } = process.env;
 
 function buildProjectURL(
   scheme: string,
   host: string,
   projectId: string,
 ): string {
-  return `${scheme}://${host}/project/${projectId}`;
+  let authority = `${scheme}://${host}`;
```

Comment:
With `const` maybe?

```js
const  authority = SERVER_URL ? SERVER_URL.replace(/\/$/, '') : `${scheme}://${host}`
```

Reply:
that's more readable? 🤔

Reply:
yes is the answer, thanks for the suggestion. Done!